homeassistant/components/nut/__init__.py

```diff
@@ -31,7 +31,7 @@
     PYNUT_UNIQUE_ID,
 )
 
-NUT_FAKE_SERIAL = ["unknown", "blank"]
+NUT_FAKE_SERIAL = ["unknown", "blank", "000000000"]
```

Comment:
It would be better to match any number of zeros

Reply:
Yeah I agree, that's a much better solution.

Reply:
Looks like they already do 
https://github.com/home-assistant/core/blob/88721a97decad341cfc15fc1ab77402a5e8f78b1/homeassistant/components/nut/__init__.py#L151

Reply:
Yup, it is indeed not needed. Well spotted @HarvsG 👍